### PR TITLE
Change Rust Hack & Learn

### PR DESCRIPTION
**Pull request by skade**
This changes the Rust activity to an instruction to set up a meetup.

It's still WIP, especially around which links to give to the organisers.

### CODE REVIEW

**Comment by skade** [comment]
@MichaelKohler clerical error, thanks for the notice.

I intend to update it before a merge, I still want to have people from the Rust content team to review it and give a few pointers to beginners material.

**Comment by skade** [comment]
@MichaelKohler @badboy @ashleygwilliams Needs final review, along with https://github.com/rust-community/events-team/pull/9/files

🚢ing ! \o/

**Comment by MichaelKohler** [comment]
I will review this later today. I'd also like @couci to review this.

File: _pages/rust-hack.md [hunk 1]
```diff
@@ -49,41 +55,44 @@ Rust is a systems programming language that runs blazingly fast, prevents segfau
...
+* Do an intro round (5 minutes). Ask everyone for:
+  - Who they are
+  - What previous programming experiences they have
+  - What they intend to work on (or if they are searching for a project)
+* Encourage people to group up or work alone
+* Introduce new people to the existing resources
+* Help people that don't have a project to find one
+* Open time: people should just work on whatever they have. Some people will just chat, make sure they don't disturb others.
+* Closing: go through the group again and ask people for their progress.
+  - If time allows, make a [Show \& Tell](https://en.wikipedia.org/wiki/Show_and_tell_(education)) and have people actually present their learnings in a very short talk.
```
Comment (MichaelKohler) [inline _pages/rust-hack.md]: This renders as "Show \\& Tell" ;)

File: _pages/id/rust-hack.md [hunk 2]
```diff
@@ -111,7 +97,7 @@ Additional to the recommended event flow here are some things we’d like you, t
     * [Here is the Discourse post to write to](https://discourse.mozilla-community.org/t/activate-mozilla-dive-into-rust/10073/1)
 
 ## Extra
-While the focus of this activity should be on these events, there are other ways to stay more involved with the Rust community after.
+While the focus of this activity should be on these events, people can get more involved more involved with the Rust community after.
```
Comment (couci) [inline _pages/id/rust-hack.md]: I guess here it's "people can get more involved with the Rust community after

File: _pages/rust-hack.md [hunk 3]
```diff
@@ -15,30 +15,36 @@ redirect_from: "/developer-engagement/rust-hack/"
 
 The Rust programming language will be important to the future of the web, making it safe and great. Having more developers understand, use and evangelize this language will advance make the web better and more secure.
 
-Rust will also be used extensively for Mozilla projects -- including this year as Servo (written in Rust) is advanced even further, and parts of it are moved into Gecko/Firefox. Having more Mozilla contributors know Rust will give them a chance to contribute to the most exciting and important technical projects coming up for Mozilla.
+Rust is used extensively for Mozilla projects -- including this year as Servo (written in Rust) is advanced even further, and parts of it are moved into Gecko/Firefox. Having more Mozilla contributors know Rust will give them a chance to contribute to the most exciting and important technical projects coming up for Mozilla.
+
+Rust is also a good project to be engaged in for people that don't want or can't participate in a Mozilla project, but like to be involved with something close.
```
[the remark this answers is not in the document]
Comment (couci) [reply _pages/rust-hack.md]: I think it's ok. While activate is organized mainly by Mission Driven Mozillians, thus their main objective is to bring more people to Mozilla, it's ok to mention that Rust is also a stand alone thing

File: _pages/rust-hack.md [hunk 4]
```diff
@@ -15,30 +15,36 @@ redirect_from: "/developer-engagement/rust-hack/"
...
 ### Goals for this area
 
 __Impact:__
 
-* 1,000 programmers/developers are introduced to and are well on their way to learning Rust
-* 10 blog posts about events or other local Rust activities
+* 1,000 programmers are introduced to and are well on their way to learning Rust
+* 10 ongoing Rust Hack and Learn events
+* 500 social media shares
+* Each organizer schedules 3 Rust Hack and Learn events
```
[the remark this answers is not in the document]
Comment (couci) [reply _pages/rust-hack.md]: I understand this as a whole activity goal and not per person

File: _pages/rust-hack.md [hunk 5]
```diff
@@ -49,41 +55,44 @@ Rust is a systems programming language that runs blazingly fast, prevents segfau
 
 ## Activity Format
 
-Get your community behind this exciting new programming language by running a learning event.
+For duration, we recommend at least 2-3 hours event with a minimum of 10 attendees.
```
[the remark this answers is not in the document]
Comment (couci) [reply _pages/rust-hack.md]: "a 2-3 hours event"

**Comment by couci** [comment]
thanks @skade for all the work here, except the minor stuff mentioned this looks good to me

File: _pages/rust-hack.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (skade) [reply _pages/rust-hack.md]: "Rust is also a good subject to get people closer to Mozilla gradually, as it has smaller project sizes and potential for individual contribution - for example by writing a useful library."

File: _pages/rust-hack.md [hunk 6]
```diff
@@ -15,30 +15,36 @@ redirect_from: "/developer-engagement/rust-hack/"
 
 The Rust programming language will be important to the future of the web, making it safe and great. Having more developers understand, use and evangelize this language will advance make the web better and more secure.
 
-Rust will also be used extensively for Mozilla projects -- including this year as Servo (written in Rust) is advanced even further, and parts of it are moved into Gecko/Firefox. Having more Mozilla contributors know Rust will give them a chance to contribute to the most exciting and important technical projects coming up for Mozilla.
+Rust is used extensively for Mozilla projects -- including this year as Servo (written in Rust) is advanced even further, and parts of it are moved into Gecko/Firefox. Having more Mozilla contributors know Rust will give them a chance to contribute to the most exciting and important technical projects coming up for Mozilla.
+
+Rust is also a good project to be engaged in for people that don't want or can't participate in a Mozilla project, but like to be involved with something close.
 
 ### Goals for this area
```
[the remark this answers is not in the document]
Comment (skade) [reply _pages/rust-hack.md]: What would the measurement period be? "In 2018?"

**Comment by skade** [comment]
Pushed a polished version, /cc @couci @MichaelKohler

File: _pages/rust-hack.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (skade) [reply _pages/rust-hack.md]: That's also how I understood it from the other activities.

**Comment by samim111** [comment]
sscccssxx

**Comment by skade** [comment]
@MichaelKohler 

> We've rewritten the Rust activity, as a lot of it was outdated. We also changed its nature a little. It now describes how to kick off a regular meeting, in the style of a "Hack & Learn". This allows you to run the event without having a speaker close by or without having a lot of Rust experience. The goals have changed a little. Another change was to get the activity a little more integrated with the Rust community, so that they can assist you with you questions and offer promotion. It should now also be easier to run and setup.

**Comment by MichaelKohler** [comment]
@skade thanks a lot!

@couci wanna do a final sign-off on this? Once done, I can merge it and then announce it on Discourse / reps-general and Telegram.

**Comment by couci** [comment]
lgtm thanks a lot @skade !

**Comment by MichaelKohler** [comment]
Thanks for the review. I guess this is ready to be merged then! I will do so and send out the communication by end of the weekend at latest.

**Pull request merged**